### ISSUE TITLE
Remove Mambaforge download links

### ISSUE THREAD

**Issue by ethanc8** [conda-forge/conda-forge.github.io#2336]
On the [download site](https://conda-forge.org/miniforge/), the first download links the user sees are Mambaforge download links. However, [Mambaforge is deprecated](https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/), so the Mambaforge download links should be removed or made less visible. Additionally, since [conda-forge's pypy support is deprecated](https://conda-forge.org/news/2024/08/14/sunsetting-pypy/), the Miniforge-pypy3 download links should also be made less visible. If we keep those download links on the page, we should add a note that they are deprecated. Perhaps they could be hidden under a collapsible element like the download links for older versions are.

**Comment by hmaarrfk** [comment]
hmm a PR to deprioritize them would be good.

I don't even know how you go there. 

I can only get to https://conda-forge.org/download/

**Comment by hmaarrfk** [comment]
Yeah.... ok as a first step. i tried to point people to the right direction.

![Image](https://github.com/user-attachments/assets/3f462a95-4792-4bbb-9c5d-3d7bf1d4eba7)

**Comment by ethanc8** [comment]
I meant that the download page hosted at https://conda-forge.org/miniforge/ is the second search result.

**Comment by hmaarrfk** [comment]
i understand, but i'm just letting you know:

1. I acknowledge the pain point.
2. I personally am unsure how to fix this.
3. I added a step in the right direction.

**Comment by ethanc8** [comment]
I believe this file contains the source for the download page: https://github.com/conda-forge/miniforge/blob/gh-pages/index.html. However, it seems to be generated by a GH Actions workflow.

**Comment by ethanc8** [comment]
I believe it is the ["Build docs"](https://github.com/conda-forge/miniforge/actions/workflows/docs.yml) action that causes this. This workflow uses the [release.py](https://github.com/conda-forge/miniforge/blob/main/docs/releases.py) script, so modifying that script and re-running the action should fix this.

**Comment by ethanc8** [comment]
At minimum, we can modify the [all-releases.html](https://github.com/conda-forge/miniforge/blob/main/docs/templates/all-releases.html) page to have a link to the README and a note that PyPy3 and Mambaforge installers are deprecated.

**Comment by hmaarrfk** [comment]
That sounds like a good improvement! 

A PR would be appreciate.

We could also pull in maybe the release description?

**Comment by ethanc8** [comment]
I might work on this later, then. I've got quite a lot to do now.
